yt_dlp/extractor/tubitv.py

```diff
@@ -119,6 +119,15 @@ def _real_extract(self, url):
             'release_year': int_or_none(video_data.get('year')),
         }
 
+        episode_match = re.search(r'^S(\d+):E(\d+) - (.+)', title)
+
+        if episode_match:
+            infodict['season_number'] = int(episode_match.group(1))
+            infodict['episode_number'] = int(episode_match.group(2))
```

Comment:
use int_or_none

Reply:
Since the match is done with `\d+`, int is fine